docs/docs/configuration/nextjs.md

```diff
@@ -1,5 +1,60 @@
 # Next.js
 
+## `getServerSession`
+
+> 🚧 This is experimental and may be removed in the future.
```

Comment:
```suggestion
> 🚧 This is experimental and may be removed or changed in the future.
```

Let's keep this hanging... :sweat_smile: Might stick around for a while. Might not...